common/models/user.js

```diff
@@ -346,6 +346,10 @@ module.exports = function(User) {
   };
 
   User.observe('before delete', function(ctx, next) {
+    
+    // Do nothing and just go to next when the user is not logged in
```

Comment:
> when the user is not logged in

This description feels misleading to me. How about this?

_Do nothing and when the access control was disabled for user model._

Reply:
Uh oh, my proposed description is not even proper English. Sorry for the typo! Here is what I meant:

_Do nothing when the access control was disabled for this user model._

Reply:
Ok, I fixed all the linting problem, and excuse my english because I'm not very english. Feel free to modify anything you want.